site/js/SessionActor.js

```diff
@@ -43,6 +46,35 @@ export default class SessionActor extends Actor {
       this.switchServer(url)
     })
 
+    let serverListOpen = false
+    document.querySelector('.server-list-heading').addEventListener('click', async evt => {
+      const serverListEl = document.getElementById('server-list')
+      serverListEl.style.setProperty('--server-list-height', ((Object.keys(this.sessionIDs).length + 1) * 54 + 32) + 'px')
```

Comment:
MAGIC NUMBERS AAAAAAAAAAAAAAAAAAAa.

Is this necessary? Why must this be set through JS? Couldn't you just use a collapsed/open class, and have it only be short when collapsed?

Reply:
>Magic numbers

no u, these are the heights of different elements as set in style.css. I guess they could do with some clarification...?

> Is this necessary? Why must this be set through JS?

Because I don't want to do the `max-height: auto; height: 99999px;` animation hack.

Reply:
>animation hack

Oh, this is for animation; that makes more sense.

This could do with some explanation, yeah. After that this is fine to merge!

Reply:
...there is animation. Right now :shipit:

Reply:
Done. Please re-review 📦